task_sdk/src/airflow/sdk/definitions/macros.py

```diff
@@ -23,6 +23,15 @@
 import uuid  # noqa: F401
 from datetime import datetime, timedelta
 from random import random  # noqa: F401
+from typing import TYPE_CHECKING, Any
+
+import dateutil  # noqa: F401
+
+import airflow.utils.yaml as yaml  # noqa: F401
```

Comment:
We should at some point figure out moving things like this from core to SDK or go for redundancy to keep it decoupled